legate/core/operation.py

```diff
@@ -112,14 +125,18 @@ def _get_unique_partition(self, store):
             )
         return parts[0]
 
-    def add_input(self, store, partition=None):
+    def add_input(
+        self, store: Store, partition: Optional[PartSym] = None
+    ) -> None:
         self._check_store(store)
         if partition is None:
             partition = self._get_unique_partition(store)
-        self._inputs.append(store)
+        self._inputs.append(store)  # XXXX
```

Comment:
What does this comment mean?

Reply:
Sorry, this was just a landmark for quicky finding the lines I wanted to show by searching on the video call, forgot to remove https://github.com/nv-legate/legate.core/pull/189/commits/7bc51b3a159364102d2a8a3fbd2049bbbe585d5c